flight/Modules/System/systemmod.c

```diff
@@ -44,7 +44,9 @@
 #include "watchdogstatus.h"
```

Comment:
The event task used to both do the work of periodic callbacks and of pumping events from a queue to user-defined callbacks.  Since this is now direct, there is much less need for it as a separate task.

Also, periodic callbacks conform to the UAVO callback signature but really aren't related.